client/my-sites/site-settings/jetpack-credentials/credentials-configured/index.jsx

```diff
@@ -34,6 +42,25 @@ class CredentialsConfigured extends Component {
 
 	toggleRevoking = () => this.setState( { isRevoking: ! this.state.isRevoking } );
 
+	supportsRealtimeBackup() {
```

Comment:
I was thinking it would be nice to avoid this duplication, but somehow it felt too concrete to have this as a separate selector. But if y'all think we should create that selector, I'm fine to go that way!

Reply:
I wonder if it may be useful to have a selector that's a more generic `planSupports` query for use cases like this. May not be something for this PR, but would be good to avoid duplication of logic like this down the road.

Reply:
I think it's best as its own selector. For example, see `has_jetpack_backup_realtime_feature` in jetpack.com's store. There are a variety of functions like this available for reuse there, and I think Calypso would benefit from something similar.

Reply:
Okay, fair enough folks! I've created a `siteSupportsRealtimeBackup` selector with some tests, and adapted the code to use that selector. Take another look when you get a chance 😉